projects/core/src/personalization/config/default-personalization-config.ts

```diff
@@ -1,5 +1,8 @@
 import { PersonalizationConfig } from './personalization-config';
 
+/**
+ * @deprecated since 3.2, use tracking/personalization instead
```

Comment:
I would rather reference the full library name (@spartacus/tracking/personalization) in those deprecations.